### ISSUE TITLE
Minimize and close button not showing on V13 when in a minimized state

### ISSUE THREAD

**Issue by CodaBool** [CodaBool/terminal#20]
# 🚩 The Issue
Minimize and close button not showing on V13 when in a minimized state

---

### 🖥️  Setup
- **Foundry Version** = 13.333
- **Game System** = Alien
- **Terminal Module Version** = 3.1.10
- **Browser** = Brave

**Comment by relaytheory** [comment]
Similar issue present on:
Foundry Version = 12.331
Game System = Pathfinder 2e
Terminal Module Version = 3.1.10
Browser = Both Chromium and Firefox

![Screenshot_6](https://github.com/user-attachments/assets/7ba4a7ab-815c-4518-9e1c-d439813e56a5)

**Comment by CodaBool** [comment]
@relaytheory oh wow that's even when it's not in a minimized state. I'll see if I can get this one fixed this weekend.

**Comment by CodaBool** [comment]
Just gave this a test and wasn't able to reproduce the issue on my end

@relaytheory If you're willing to contribute the following would be helpful:

- Steps needed to reproduce the issue
- Does it still do this when all other modules are disabled?
- Are there any relevant logs in the [dev console](https://balsamiq.com/support/faqs/browserconsole) (F12 > console) ?

**Comment by relaytheory** [comment]
Heya!
Here is a quick video of me performing the entire process from start to finish.
The journal itself was made a few seconds prior, but was not captured.
If you need any more information, feel free to reach out.

https://github.com/user-attachments/assets/5e55a10a-6c29-4e26-bbd9-699ed316da52

**Comment by CodaBool** [comment]
something seems off about how the titlebar is not showing at the loading screen too. I think that used to be the behavior a couple versions ago. 

I think the next best troubleshooting would be to uninstall and reinstall the module. My guess here is that you are getting an old file for either the styling (.css) or the template file (.hbs) for some weird reason.

There is a way to peak into the files under the dev tools console under the Sources tab (`assets.forge-vtt.com/bazaar/modules/terminal-xxxxxxxxxxxxxxx/3.1.1x-xxxxxxxxxxxx/scripts/hooks.js`). Then we could know for sure if you have old files somehow. But I don't want to complicate things too early.

Just do a reinstall and let me know how that goes. 

Also since you're on Forge. There are two ways you can install/reinstall. You can do a "Return to Setup" from in the game and then manage the module. Or you could use the [Bazaar](https://forge-vtt.com/bazaar/package/terminal). I'd recommend trying the other if one does not fix the issue.

Hopefully that fixes it.

**Comment by relaytheory** [comment]
So weird thing.
A complete reinstall of v3.1.10 through the Bazaar did not work.
Then I updated to v3.1.11, and now it works.
Lol.
Consider the v12 issue fixed?

**Comment by CodaBool** [comment]
Yep, pretty strange, but glad you're all set now. Closing this out.

🌞

**Comment by relaytheory** [comment]
Small addendum:
The issue reappeared, but it has been identified as a compatibility issue with Dorako UI + Doraku UX.
There is likely some alterations made by Dorako UI to journals that conflicts with Terminal.

It occurs when any changes are made to the Sheet theme.
![Screenshot_6](https://github.com/user-attachments/assets/a08be5e6-d9c4-48e9-bbb5-3a26bb4de23c)

**Comment by CodaBool** [comment]
Oh neat that's great info

I'll get a fix out sometime within a week.

**Comment by CodaBool** [comment]
Looks like I actually had a styling bug with Light theme in general. I fixed it in `3.1.12` but there is still a `background` styling conflict with `Dorako UI` & `Doraku UX`. For now you can just add `Terminal` to the exclude list in Dorako UI `excluded applications` settings. But I'm working with the dev to get it added automatically.

**Comment by CodaBool** [comment]
sent the maintainer another ping

**Comment by CodaBool** [comment]
looks like it Terminal is now excluded automatically. 
That should be everything.
